audio/filter/af_scaletempo2.c

```diff
@@ -8,7 +8,7 @@
 #include "options/m_option.h"
 
 struct priv {
-    struct mp_scaletempo2 data;
```

Comment:
I changed this to a pointer so it can be used as a talloc context, so a lot of dereferences are added. Is there a better way?

Reply:
You can use `priv` (`p`) as a parent it is already allocated and suitable as talloc context.

Reply:
I was thinking of the context used in scaletempo2_internal.c. `priv` isn't available there, only the `struct mp_scaletempo2 data` reference is.